src/components/Call/Call.js

```diff
@@ -24,38 +26,24 @@ export default function Call() {
     }, []),
   );
 
-  /* This is for displaying our self-view. */
-  const localParticipant = useLocalParticipant();
-  const localParticipantVideoTrack = useVideoTrack(localParticipant?.session_id);
-  const localVideoElement = useRef(null);
-
-  useEffect(() => {
-    if (!localParticipantVideoTrack.persistentTrack) return;
-    localVideoElement?.current &&
-      (localVideoElement.current.srcObject =
-        localParticipantVideoTrack.persistentTrack &&
-        new MediaStream([localParticipantVideoTrack?.persistentTrack]));
-  }, [localParticipantVideoTrack.persistentTrack]);
-
   /* This is for displaying remote participants: this includes other humans, but also screen shares. */
   const { screens } = useScreenShare();
   const remoteParticipantIds = useParticipantIds({ filter: 'remote' });
 
+  /* This is for displaying our self-view. */
+  const localParticipant = useLocalParticipant();
+  const isAlone = useMemo(
+    () => remoteParticipantIds?.length < 1 || screens?.length < 1,
+    [remoteParticipantIds, screens],
+  );
+
   const renderCallScreen = () => {
     return (
       <div className={`${screens.length > 0 ? 'is-screenshare' : 'call'}`}>
         {/*Your self view*/}
         {localParticipant && (
-          <div
-            className={
-              remoteParticipantIds?.length > 0 || screens?.length > 0
-                ? 'self-view'
-                : 'self-view alone'
-            }>
-            <video autoPlay muted playsInline ref={localVideoElement} />
-            <div className="username">
-              {localParticipant?.user_name || localParticipant?.user_id} (you)
-            </div>
+          <div className={isAlone ? 'self-view alone' : 'self-view'}>
```

Comment:
Do you think it might make sense to move this CSS from Call.js to Tile.js? Because now that I think about it this is more Tile logic than Call logic 🤔

Reply:
Maybe something along the lines of:

```
  <div className={`${screens.length > 0 ? 'is-screenshare' : 'call'}`}>
        {/*Your self view*/}
          <Tile
            isLocalParticipant
            isAlone={isalone}
          />
        {/*Videos of remote participants and screen shares*/}
        ...more tiles...
</div>
```

Reply:
sure, sounds good!

Reply:
I moved some of the styles but not everything. Trying to not get deep in CSS changes since this is just to fix the flickering!